proto/proto/view.proto

```diff
@@ -101,6 +101,8 @@ message NoteRecord {
     optional uint64 height_spent = 6;
     // The note position.
     uint64 position = 7;
+    // Note quarantine status
+    bool quarantined = 8;
```

Comment:
This should record the unbonding epoch. Otherwise, there's no way for a client to know when the quarantined notes will be released.